src/includes/class-alma-wc-payment-gateway.php

```diff
@@ -757,4 +788,87 @@ private function get_fee_plan_definition( $plan_key ) {
 
 		return $definition;
 	}
+
+	/**
+	 * AJAX when validating the checkout.
+	 * If the payment method used is like "alma_****", then rename it to "alma" and let WC do the payment process.
+	 *
+	 * @return void
+	 */
+	public function woocommerce_checkout_process() {
+		if ( substr( $_POST['payment_method'], 0, 5 ) === 'alma_' ) { // phpcs:ignore
+			$_POST['payment_method'] = 'alma';
+		}
+	}
+
+	/**
+	 * Filter available_gateways to add "alma_pay_later" and "alma_more_than_four_instalments".
+	 *
+	 * @param array $_available_gateways The list of available gateways.
+	 * @author Gilles Dumas
+	 * @return array
+	 */
+	public function woocommerce_available_payment_gateways( $_available_gateways ) {
```

Comment:
refactor this method to reduce [cognitive complexity](https://sonarcloud.io/project/issues?id=alma_alma-woocommerce-gateway&open=AX6bUcQ20-qjSFvPRM4M&pullRequest=36&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL)